internal/ethapi/api.go

```diff
@@ -1822,13 +1823,26 @@ func (s *TransactionAPI) FillTransaction(ctx context.Context, args TransactionAr
 	return &SignTransactionResult{data, tx}, nil
 }
 
+// change(taiko): preconf logic
 // SendRawTransaction will add the signed transaction to the transaction pool.
 // The sender is responsible for signing the transaction and using the correct nonce.
-func (s *TransactionAPI) SendRawTransaction(ctx context.Context, input hexutil.Bytes) (common.Hash, error) {
+func (s *TransactionAPI) SendRawTransaction(ctx context.Context, txBytes hexutil.Bytes, slot *uint64, signature *string) (interface{}, error) {
+	// Decode the raw transaction
 	tx := new(types.Transaction)
-	if err := tx.UnmarshalBinary(input); err != nil {
+	if err := tx.UnmarshalBinary(txBytes); err != nil {
 		return common.Hash{}, err
 	}
+
+	fmt.Println("url", s.b.GetPreconfirmationForwardingURL(), slot, signature)
```

Comment:
Please move this line to line1840 and change to use `log.Info`.

Reply:
debug stuff is just staying in for a bit until we want to see if this is how we want to go forward then I'll clean it up.

Reply:
Ok, get it.